docs/ref/contrib/postgres/functions.txt

```diff
@@ -0,0 +1,26 @@
+PostgreSQL specific database functions
+======================================
+
+All of these fields and widgets are available from the
+``django.contrib.postgres.functions`` module.
+
+.. currentmodule:: django.contrib.postgres.functions
+
+TransactionNow
+--------------
+
+.. class:: TransactionNow()
+
+.. versionadded:: 1.9
+
+Returns the date and time on the database server that the current transaction
+started. If you are not in a transaction it will return the date and time of
+the current statement. This is a complement to
+:class:`django.db.models.functions.Now`, which always returns the date and time
+of the current statement.
```

Comment:
If we're inside several nested transactions, will it returns the time of the top or the bottom one ?